client/web/src/enterprise/batches/list/BatchChangesChangelogAlert.tsx

```diff
@@ -8,22 +8,21 @@ import styles from './BatchChangesListIntro.module.scss'
 export const BatchChangesChangelogAlert: React.FunctionComponent = () => (
     <DismissibleAlert
         className={styles.batchChangesListIntroAlert}
-        partialStorageKey="batch-changes-list-intro-changelog-3.35"
+        partialStorageKey="batch-changes-list-intro-changelog-3.36"
     >
         <div className={classNames(styles.batchChangesListIntroCard, 'card h-100 p-2')}>
             <div className="card-body">
-                <h4>Batch Changes updates in version 3.35</h4>
+                <h4>Batch Changes updates in version 3.36</h4>
                 <ul className="mb-0 pl-3">
                     <li>
                         <a
-                            href="https://docs.sourcegraph.com/batch_changes/references/batch_spec_yaml_reference#on-repository"
+                            href="https://docs.sourcegraph.com/admin/config/batch_changes#forks"
                             rel="noopener"
                             target="_blank"
                         >
-                            Batch Changes now supports multiple base branches per repository.
+                            Batch Changes now supports forks for changesets.
```

Comment:
Slight rewording suggestions:

```suggestion
                            Batch Changes now supports pushing changesets to forked repositories.
```

Reply:
I tweaked this 2 or 3 times already before I pushed it and settled on just "forks", because we also support repos not yet forked and such.. but not a strong preference :) I'll keep this up until tomorrow and see if other people have opinions, otherwise I'll go with your suggestion.

Reply:
I guess my vote is for Adam's wording, since it parallels the first sentence in the description on the docs!